xml/System/SequencePosition.xml

```diff
@@ -192,8 +192,8 @@
       </ReturnValue>
```

Comment:
What's the expectation for things like GetHashCode which are documented as `inheritdoc`?
https://github.com/dotnet/corefx/blob/7f21952eed6c421d6854bcce3c0a8f76f2d5de9b/src/System.Memory/src/System/SequencePosition.cs#L53

Looks like we need to add something here for the right thing to get displayed:
https://docs.microsoft.com/en-us/dotnet/api/system.sequenceposition?view=netstandard-2.1